Modules/Bidding.lua

```diff
@@ -339,84 +348,70 @@ function CommDKP:WhisperAvailableDKP(name, cmd)
+    return from.Belt
   elseif loc == "INVTYPE_LEGS" then
-    return core.DB.MinBidBySlot.Legs
+    return from.Legs
   elseif loc == "INVTYPE_FEET" then
-    return core.DB.MinBidBySlot.Boots
+    return from.Boots
   elseif loc == "INVTYPE_FINGER" then
-    return core.DB.MinBidBySlot.Ring
+    return from.Ring
   elseif loc == "INVTYPE_TRINKET" then
-    return core.DB.MinBidBySlot.Trinket
+    return from.Trinket
   elseif loc == "INVTYPE_WEAPON" or loc == "INVTYPE_WEAPONMAINHAND" or loc == "INVTYPE_WEAPONOFFHAND" then
-    return core.DB.MinBidBySlot.OneHanded
+    return from.OneHanded
   elseif loc == "INVTYPE_2HWEAPON" then
-    return core.DB.MinBidBySlot.TwoHanded
+    return from.TwoHanded
   elseif loc == "INVTYPE_HOLDABLE" or loc == "INVTYPE_SHIELD" then
-    return core.DB.MinBidBySlot.OffHand
+    return from.OffHand
   elseif loc == "INVTYPE_RANGED" or loc == "INVTYPE_THROWN" or loc == "INVTYPE_RANGEDRIGHT" or loc == "INVTYPE_RELIC" then
-    return core.DB.MinBidBySlot.Range
+    return from.Range
   else
-    return core.DB.MinBidBySlot.Other
+    return from.Other
   end
 end
 
-function CommDKP:GetMaxBid(itemLink)
-  local _,_,_,_,_,_,_,_,loc = GetItemInfo(itemLink);
+function CommDKP:GetMinBid(itemLink)
+  local itemPrice = CommDKP:GetItemPrice(itemLink);
+  if itemPrice then
+    return itemPrice.minBid
+  end
 
-  if loc == "INVTYPE_HEAD" then
-    return core.DB.MaxBidBySlot.Head
-  elseif loc == "INVTYPE_NECK" then
-    return core.DB.MaxBidBySlot.Neck
-  elseif loc == "INVTYPE_SHOULDER" then
-    return core.DB.MaxBidBySlot.Shoulders
-  elseif loc == "INVTYPE_CLOAK" then
-    return core.DB.MaxBidBySlot.Cloak
-  elseif loc == "INVTYPE_CHEST" or loc == "INVTYPE_ROBE" then
-    return core.DB.MaxBidBySlot.Chest
-  elseif loc == "INVTYPE_WRIST" then
-    return core.DB.MaxBidBySlot.Bracers
-  elseif loc == "INVTYPE_HAND" then
-    return core.DB.MaxBidBySlot.Hands
-  elseif loc == "INVTYPE_WAIST" then
-    return core.DB.MaxBidBySlot.Belt
-  elseif loc == "INVTYPE_LEGS" then
-    return core.DB.MaxBidBySlot.Legs
-  elseif loc == "INVTYPE_FEET" then
-    return core.DB.MaxBidBySlot.Boots
-  elseif loc == "INVTYPE_FINGER" then
-    return core.DB.MaxBidBySlot.Ring
-  elseif loc == "INVTYPE_TRINKET" then
-    return core.DB.MaxBidBySlot.Trinket
-  elseif loc == "INVTYPE_WEAPON" or loc == "INVTYPE_WEAPONMAINHAND" or loc == "INVTYPE_WEAPONOFFHAND" then
-    return core.DB.MaxBidBySlot.OneHanded
-  elseif loc == "INVTYPE_2HWEAPON" then
-    return core.DB.MaxBidBySlot.TwoHanded
-  elseif loc == "INVTYPE_HOLDABLE" or loc == "INVTYPE_SHIELD" then
-    return core.DB.MaxBidBySlot.OffHand
-  elseif loc == "INVTYPE_RANGED" or loc == "INVTYPE_THROWN" or loc == "INVTYPE_RANGEDRIGHT" or loc == "INVTYPE_RELIC" then
-    return core.DB.MaxBidBySlot.Range
-  else
-    return core.DB.MaxBidBySlot.Other
+  return CommDKP:GetItemLocBid(core.DB.MinBidBySlot, itemLink)
+end
+
+function CommDKP:GetMaxBid(itemLink)

```

Comment:
You can see it added here again. It's just the diff tool.

Basically I just refactored the two methods to reduce the amount of code.